doc/parent_child_spec.mld

```diff
@@ -13,9 +13,9 @@ The rules are;
 {- The output paths of [.mld] files and compilation units are subdirectories of their parent's
  output directory.}
 {- The output directory of a [.mld] file [x.mld] with children is [<parent_output_directory>/x],
-   and its file name is [index.html]. That is to say, [<parent_output_directory>/x/index.html]}
-{- The output directory of a [.mld] file [x.mld] without children is [<parent_output_directory> /x.html]}
-{- The output directory of a compilation unit [X] is [<parent_output_directory>/X/index.html]}
+   and its file name is [index.html]. That is to say, [<parent_output_directory>/x/index.html].}
+{- The output directory of a [.mld] file [x.mld] without children is [<parent_output_directory> /x.html].}
```

Comment:
Is `.mld` pronounced "em el dee" or "dot em el dee"? If the former, the article should be "an," so I'll make sure those are all consistent. I believe in other cases, the "dot" isn't pronounced, so let me know!

Reply:
Sorry, no idea! I guess several person are pronouncing it differently... even myself do not pronounce it always the same.

I'll be happy with any convention!